src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -3023,19 +3148,19 @@ export class StudyViewPageStore {
         default: []
     });
 
-    readonly cnaGeneData = remoteData<CopyNumberCountByGeneWithCancerGene[]>({
+    readonly cnaGeneData = remoteData<GeneTableRow[]>({
```

Comment:
Is this a TableRow in the sense of a database row or a <html> table row?    If the latter, i think name of the promise (cnaGeneData) should reflect that.  "cnaGeneTableRowData"